docs/src/pages/NativePickerPage/NativePickerPage.js

```diff
@@ -65,6 +63,12 @@ const NativePickerSubPage = ({ ...rest }) => (
     components={components}
     readme={readme}
     showMenu
+    usageTable={{
+      dos: ['Use for short lists of options.'],
+      donts: [
+        'Use for lots of options where the user may need to scroll a lot, for example a list of countries. In this situation, consider using a FlatList.',
```

Comment:
Might be worth linking to the FlatList component here?

Reply:
It only accepts strings, so I thought let's just fix that later, seeing as we'll probably move to a 'proper' guidelines thing soon that's more advanced than this. But good spot!